lib/logstash/outputs/scalyr.rb

```diff
@@ -400,6 +408,23 @@ def multi_receive(events)
   end  # def multi_receive
 
 
+  def log_retry_failure(multi_event_request, exc_data, exc_retries, exc_sleep)
+    message = "Failed to send event after #{exc_retries} tries."
```

Comment:
I guess this should be "events" (plural), right? And maybe also not a big idea to include number of events in batch.

For example: "Failed to send Y events to Scalyr after X retries".